File: Exemplos SmartPOS/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_SmartPOS_Flutter-font/lib/IntentDigitalHubService/ELGINPAY/elginpay_command.dart
import 'package:flutter_smartpos/IntentDigitalHubService/intent_digital_hub_module.dart';
import '../intent_digital_hub_command_starter.dart';

abstract class ElginPayCommand extends IntentDigitalHubCommand {
  ElginPayCommand(String functionName)
      : super(functionName, IntentDigitalHubModule.ELGINPAY);
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_SmartPOS_Ionic-font/src/intentDigitalHubService/ELGINPAY/ElginPayCommand.ts
import {IntentDigitalHubCommand} from '../IntentDigitalHubCommand';
import {IntentDigitalHubModule} from '../IntentDigitalHubModule';

/**
 * Classe que generaliza todos os comandos do módulo ELGINPAY, definindo o módulo do comando e o seu tipo de retorno
 */
export abstract class ElginPayCommand extends IntentDigitalHubCommand {
  constructor(functionName: string) {
    super(functionName, IntentDigitalHubModule.ELGINPAY);
  }
}
